t/t0003-api-client.t

```diff
@@ -0,0 +1,76 @@
+#!/bin/sh
+
+test_description='Test simple Powerman API client'
+
+. `dirname $0`/sharness.sh
+
+powermand=$SHARNESS_BUILD_DIRECTORY/src/powerman/powermand
+powerman=$SHARNESS_BUILD_DIRECTORY/src/powerman/powerman
+test_apiclient=$SHARNESS_BUILD_DIRECTORY/src/powerman/test_apiclient
+vpcd=$SHARNESS_BUILD_DIRECTORY/test/vpcd
+vpcdev=$SHARNESS_TEST_SRCDIR/../etc/devices/vpc.dev
+
+testaddr=localhost:10104
+
+test_expect_success 'create test powerman.conf' '
+	cat >powerman.conf <<-EOT
+	include "$vpcdev"
+	listen "$testaddr"
+	device "test0" "vpc" "$vpcd |&"
+	node "t[0-15]" "test0"
+	EOT
+'
+
+test_expect_success 'start powerman daemon and wait for it to start' '
+	$powermand -c powerman.conf -f &
+	echo $! >powermand.pid &&
+	$powerman --retry-connect=10 --server-host=$testaddr -q >/dev/null
+'
+
+test_expect_success 'API query works' '
+	$test_apiclient $testaddr q t1 >query.out &&
+	cat >query.exp <<-EOT &&
+	t1: off
+	EOT
+	test_cmp query.exp query.out
+'
```

Comment:
not necessarily for this PR, but it seems weird that you check t1 for "off", turn on "t0", then check "t0" is on.  Would it make more sense to "check t0 off, turn on t0, check to on"?